### PR TITLE
DDF 3299 2.11.x

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Reorganizes configuring instructions of documentation to follow task-based format more closely. 
Added TODOs for follow-on work to draft new sections. 
The only new content in this PR is small intro sections to tie together the existing content.

#### Who is reviewing it? 
@mcalcote @vinamartin @ahoffer @garrettfreibott @peterhuffer @blen-desta @djblue 

#### Choose 2 committers to review/merge the PR. 

@clockard
@coyotesqrl
@lessarderic
@pklinef
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- Builds successfully
- Relevant sections being included
- Outline functioning appropriately. (Check the Table of contents on the left margin of the html version)

#### What are the relevant tickets?
[DDF-3299](https://codice.atlassian.net/browse/DDF-3299)

#### Screenshots (if appropriate)

[documentation.zip](https://github.com/codice/ddf/files/1367130/documentation.zip)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_configuring/admin-wizard-placeholder.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+:type: configuration
+:status: published
+:parent: Admin Console Tutorial
+:order: 00
+:summary: Placeholder for review. To be replaced.
+////
+== {title}
+
+TODO: https://codice.atlassian.net/browse/DDF-2193 - update documentation for how to configure to use an LDAP server
+////
```
Comment (blen-desta) [inline distribution/docs/src/main/resources/content/_configuring/admin-wizard-placeholder.adoc]: ❓ Is this where the new wizards documentations will live when they're merged into DDF or is it just how to configure LDAP?

File: distribution/docs/src/main/resources/content/_configuring/connecting-to-sources.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+:title: Connecting to Sources
+:type: configuration
+:status: draft
+:parent: Configuring Federation
+:order: 01
```
Comment (blen-desta) [inline distribution/docs/src/main/resources/content/_configuring/connecting-to-sources.adoc]: ❓ Will this file and `connecting-to-registry.adoc` have different orders when they get published?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/connecting-to-sources.adoc]: Yes.

File: distribution/docs/src/main/resources/content/_configuring/admin-wizard-placeholder.adoc (hunk 1 above)
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/admin-wizard-placeholder.adoc]: This file is for the wizards specifically, but that ticket number was the closest I had for it.

**Comment by ricklarsen** [comment]
build now

File: distribution/docs/src/main/resources/content/_configuring/decanter-services.adoc [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-:order: 14
+:parent: Miscellaneous Configurations
+:summary: Configuring alerts through Decanter services.
+:order: 02
 
-==== Configuring Decanter Services By Configuration file
+== {title}
 
-By default, ${branding} uses two services provided by Decanter that can be configured by configuration file.
+By default, ${branding} uses two services provided by Decanter for alerts that can be configured by configuration file.
```
Comment (mcalcote) [inline distribution/docs/src/main/resources/content/_configuring/decanter-services.adoc]: Should we call this `Karaf Decanter`  in each of the places it is used?

Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_configuring/decanter-services.adoc]: Probably fixed

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**